ibm/mas_devops/roles/suite_app_config/vars/customspecs/manage_components.yml.j2

```diff
@@ -0,0 +1,118 @@
+strategize:
+  version: {{ manageWorkspaceComponents['strategize']['version'] }}
+{% endif %}
+# -----------------------------------------------------------------------------
+{% if manageWorkspaceComponents['transportation'] is defined and manageWorkspaceComponents['transportation']['version'] is defined %}
+transportation:
+  version: {{ manageWorkspaceComponents['transportation']['version'] }}
+{% endif %}
+# -----------------------------------------------------------------------------
+{% if manageWorkspaceComponents['tririga'] is defined and manageWorkspaceComponents['tririga']['version'] is defined %}
+tririga:
+  version: {{ manageWorkspaceComponents['tririga']['version'] }}
+{% endif %}
+# -----------------------------------------------------------------------------
+{% if manageWorkspaceComponents['utilities'] is defined and manageWorkspaceComponents['utilities']['version'] is defined %}
+utilities:
+  version: {{ manageWorkspaceComponents['utilities']['version'] }}
+{% endif %}
+# -----------------------------------------------------------------------------
+{% if manageWorkspaceComponents['workday'] is defined and manageWorkspaceComponents['workday']['version'] is defined %}
+workday:
+  version: {{ manageWorkspaceComponents['workday']['version'] }}
+{% endif %}
+# -----------------------------------------------------------------------------
+
+{% if manageWorkspaceComponents['aviation'] is defined and manageWorkspaceComponents['aviation']['version'] is defined %}
+aviation:
+{% if ibm_mas_manage_aviation_pod_templates is defined %}
+  podTemplates: {{ ibm_mas_manage_aviation_pod_templates }}
+{% endif %}
+  version: {{ manageWorkspaceComponents['aviation']['version'] }}
+{% endif %}
+# -----------------------------------------------------------------------------
+{% if manageWorkspaceComponents['acm'] is defined and manageWorkspaceComponents['acm']['version'] is defined %}
+acm:
+{% if ibm_mas_manage_acm_pod_templates is defined %}
+  podTemplates: {{ ibm_mas_manage_acm_pod_templates }}
+{% endif %}
+  version: {{ manageWorkspaceComponents['acm']['version'] }}
+{% endif %}
+# -----------------------------------------------------------------------------
+{% if manageWorkspaceComponents['civil'] is defined and manageWorkspaceComponents['civil']['version'] is defined %}
+civil:
+{% if ibm_mas_manage_imagestitching_pod_templates is defined %}
+  podTemplates: {{ ibm_mas_manage_imagestitching_pod_templates }}
+{% endif %}
+  version: {{ manageWorkspaceComponents['civil']['version'] }}
+{% endif %}
+# -----------------------------------------------------------------------------
+{% if manageWorkspaceComponents['icd'] is defined and manageWorkspaceComponents['icd']['version'] is defined %}
+icd:
+{% if ibm_mas_manage_slackproxy_pod_templates is defined %}
+  podTemplates: {{ ibm_mas_manage_slackproxy_pod_templates }}
```

Comment:
What is slackproxy? Is that an externally known name for ICD? google search shows nothing for "maximo slackproxy"

Reply:
slackproxy is the CR name defined by ICD team, we have patten to get CR name so that's why we have ibm_mas_manage_slackproxy_pod_templates variable.